x-pack/plugins/security_solution/public/management/components/artifact_list_page/components/artifact_delete_modal.tsx

```diff
@@ -139,6 +143,7 @@ export const ArtifactDeleteModal = memo<DeleteArtifactModalProps>(
             color="danger"
             onClick={onConfirm}
             isLoading={isDeleting}
+            isDisabled={isDeleting}
```

Comment:
I believe the button is also disabled when it is loading. `isDisabled` is redundant here.

Reply:
interesting. I did not know that. given that the result is the same I'll leave it as is for now

Reply:
Yeah the result is the same 👍